RELEASE/scripts/autoscend/combat/auto_combat_fall_of_the_dinosaurs.ash

```diff
@@ -0,0 +1,33 @@
+//Path specific combat handling for Fall of the Dinosaurs
+
+void fotd_combat_helper()
```

Comment:
Do you need this function at all? Couldn't you just check enemy.random_modifiers when you need to change logic based off the dino?

Reply:
possibly? I was modelling it off of disguises delimit and ocrs, both of which have a similar function.

Reply:
my assumption was that they were this way so that we ping once to check the modifier, and then dont need to ping again every time we want to make logic based off of it?

Reply:
i'd see if you can just do like `monster.random_modifiers contains "velociraptor"` instead of making another property